docs/quickstart_guide/index.rst

```diff
@@ -0,0 +1,249 @@
+        example_motor
+
+For this guide, we will launch the example_motor:
+
+::
+
+    $ lewis -k lewis.examples example_motor
+    INFO lewis.DeviceBase: Creating device, setting up state machine
+    INFO lewis.Simulation: Changed cycle delay to 0.1
+    INFO lewis.Simulation: Changed speed to 1.0
+    INFO lewis.Simulation: Starting simulation
+    INFO lewis.AdapterCollection: Connecting device interface for protocol 'stream'
+    INFO lewis.ExampleMotorStreamInterface.StreamServer: Listening on 0.0.0.0:9999
+
+The example motor is a TCP Stream device and listens on port 9999 on all adapters by default.
+
+
+Connect to Motor via Telnet
+===========================
+
+With the last command from the previous section still running, open another terminal window.
+
+Since the example motor conveniently uses CRLF line terminators, we can use telnet to talk to it:
+
+::
+
+    $ telnet localhost 9999
+    Trying 127.0.0.1...
+    Connected to localhost.
+    Escape character is '^]'.
+
+You're now connected to the TCP Stream interface of the example motor device. It understands the following commands:
+
+=======  =============
+Command  Meaning
+=======  =============
+S?       get status
+P?       get position
+T?       get target
+T=%f     set target
+H        stop movement
+=======  =============
+
+You can get more details, and details on the interface of any device, by using the ``-i`` or ``--show-interface`` argument:
+
+::
+
+    $ lewis -k lewis.examples example_motor -i
+
+Note that the commands are case sensitive. Try entering a few commands in the Telnet session:
+
+::
+
+    S?
+    idle
+    P?
+    0.0
+    T=20.0
+    T=20.0
+    S?
+    moving
+    P?
+    9.106584
+
+See `the source code <https://github.com/DMSC-Instrument-Data/lewis/blob/master/src/lewis/examples/example_motor/__init__.py>`_ of the example motor if you want to see what makes it tick.
+
+
+Connect to Motor via Control Client
+===================================
+
+In addition to the simulated TCP Stream interface, Lewis provides a so-called Control Server interface, which allows you to bypass the normal device protocol and access both device and simulation parameters directly while the simulation is running. This can be very useful for debugging and diagnostics, without having to modify the main device interface.
+
+Remote access is disabled by default and enabled only if you provide the ``-r`` argument when starting Lewis. Stop the previously launched instance of Lewis by pressing ``Ctrl-C`` and run Lewis again with the ``-r`` parameter to enable remote access like this:
+
+::
+
+    $ lewis -r localhost:10000 -k lewis.examples example_motor
+
+Lewis ships with a Control Client commandline tool that allows you to connect to it. It also has an ``-r`` argument but for the client it defaults to ``localhost:10000``, which is why it is recommended to use the same value above. Leave the Telnet session running and connected and open a third terminal.
```

Comment:
When Lewis is restarted to enable the remote access, telnet session is killed. One will have to restart the telnet. The line here indicates that the telnet session is still running.

Reply:
Nice catch! The instructions were to start lewis with the `-r` to begin with before, so this was left over from when you didn't need to restart it.